src/Components/Labels/CategoryLabel.js

```diff
@@ -34,6 +34,7 @@ const CategoryLabel = ({ tags }) => {
           icon={CATEGORY_ICONS[tag]}
           variant="outline"
           color="blue"
+          isCompact="true"
```

Comment:
```suggestion
          isCompact
```

Reply:
... also, in your version, you are passing the string with the value "true", but not a boolean. If you want to pass a boolean, you would need to use `isCompact={true}` or just `isCompact` instead.

Reply:
True! Thanks